libs/libcommon/src/libcommon/operations.py

```diff
@@ -26,6 +26,7 @@ def update_dataset(
     hf_token: Optional[str] = None,
     force: bool = False,
     priority: Priority = Priority.NORMAL,
+    skip_check_support: bool = False,
```

Comment:
We have recently had a discussion in `datasets` about parameters with reversed semantic meaning...

What about reversing the logic? Something like `do_check_support` and being True by default?

Reply:
Or `support_checking`...

Reply:
OK, you're right

Reply:
done with https://github.com/huggingface/datasets-server/pull/720/commits/52d2cf6e58c15443692fd53577408daaf636ad0f